src/components/OverlayTooltip/index.js

```diff
@@ -17,8 +17,8 @@ const OverlayTooltip = ({ id, tooltip, placement = 'left', children, ...rest })
 }
 OverlayTooltip.propTypes = {
   id: PropTypes.string.isRequired,
-  tooltip: PropTypes.string.isRequired,
-  placement: PropTypes.string,
+  tooltip: PropTypes.oneOfType([ PropTypes.string, PropTypes.node ]).isRequired,
+  placement: PropTypes.oneOf(['top', 'right', 'bottom', 'left', false]),
```

Comment:
This prop type is repeated many times. Perhaps it's worth to create a tool-tip-alignment-prop-type and re-use it.

Reply:
I agree, I changed that according to your review